voluncheer/profiles/urls.py

```diff
@@ -1,14 +1,27 @@
 from django.urls import path
-from django.urls import re_path
 
 from profiles.views.activate_email import activate
+from profiles.views.home import SignUpView
 from profiles.views.home import home
+from profiles.views.organizations import OrganizationSignUpView
 from profiles.views.profile import ProfileView
 from profiles.views.profile import profile_update
+from profiles.views.volunteers import VolunteerSignUpView
 
 urlpatterns = [
     path("", home, name="home"),
-    re_path(r"^profile/$", ProfileView.as_view(), name="profile"),
-    re_path(r"^profile/update/$", profile_update, name="profile_update"),
+    path("profile/", ProfileView.as_view(), name="profile"),
```

Comment:
These are changes already staged in Doug's PR.

Reply:
This is not in new PR.